.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -1,20 +1,14 @@
 ---
 name: Feature request
-about: Suggest an idea for this project
+about: Suggest an idea of how to improve Editor.js
 title: "\U0001F4A1"
 labels: feature
 assignees: ''
 
 ---
 
-**Is your feature request related to a problem? Please describe.**
-A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
+1. Describe a problem.
```

Comment:
problem -> feature/goal

Reply:
The feature is the solution. It will be described in the next section. In this section, it would be useful to hear the pain or a problem user reached

Reply:
okay